services/LocalStorageService.js

```diff
@@ -1,52 +1,47 @@
 /**
- * Class represents the local storage service. It implements methods needed for communication
- * with local storage.
+ * Gets the item from local storage if it is defined.
  *
- * @class LocalStorageService
+ * @static
+ * @param {string} itemName Item's name
+ * @returns value associated with item name or null if local storage is not defined
  */
-class LocalStorageService {
-  /**
-   * Gets the item from local storage if it is defined.
-   *
-   * @static
-   * @param {string} itemName Item's name
-   * @returns value associated with item name or null if local storage is not defined
-   * @memberof LocalStorageService
-   */
-  static get(itemName) {
-    if (typeof localStorage !== 'undefined') {
-      return JSON.parse(localStorage.getItem(itemName));
+export function getLocalStorageItem(itemName) {
+  if (typeof localStorage !== 'undefined') {
+    const value = localStorage.getItem(itemName);
+    if (value == null || value === 'undefined' || value === 'null') {
```

Comment:
And what if somebody really wanted to save the value "null"? 😅

Reply:
This is a get function - it would return null as expected